deploy/kubernetes/overlays/prow-gke-release-staging-rc/controller_add_snapshotter.yaml

```diff
@@ -0,0 +1,17 @@
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-pd-controller
+spec:
+  template:
+    spec:
+      containers:
+        - name: csi-snapshotter
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/csi-snapshotter:v2.1.0
```

Comment:
this should be `gke.gcr.io/csi-snapshotter:v2.1.0-gke.0`

Reply:
sorry, `gcr.io/gke-release-staging`

Reply:
Done